src/unittest/python/timer_tests.py

```diff
@@ -0,0 +1,43 @@
+from unittest import TestCase, mock
+from vyu import timer
```

Comment:
This is not really that important but, maybe move the file import down one line?

Reply:
Addressed by commit 897d23cebf0c6cc06cf33cd1def642ea07240648